python_frank_energie/models.py

```
@@ -264,82 +1468,933 @@ def today(self) -> list[Price]:
+            },
+            'gas': {
+                'mean': gas_mean,
+                'min': gas_min,
+                'max': gas_max
+            }
+        }
+
+    def calculate_stats(data):
+        print(data)
+        electricity_prices = data.MarketPrices.electricity
+        gas_prices = data.MarketPrices.gas
```

Ruff

Local variable `gas_prices` is assigned to but never used

Remove assignment to unused variable `gas_prices`

(F841)